wiki/Glossary/en.md

```diff
@@ -853,7 +715,7 @@ _osu!_ currently has three rankings:
   Scores achieved from ranked beatmaps are taken into account.
```

Comment:
ln 704: `is usually the goal of beatmappers` or something, needs a reword imo

This line: Does anyone even check ranked score leaderboards anymore